courses-app/src/common/Button/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import "./Button.css";
 const Button = (props) => {
-  const { text, onClick } = props;
+  const { text, onClick, disabled } = props;
```

Comment:
Please make the object destruction in the Component argument to keep one code style along the project